#goal definition
float64 duration
float32 angular_speed
float32 linear_speed
---
#result definition
bool completed
float32 angular_speed
float32 linear_speed
---
#feedback definition
float64 current_time
float32 angular_speed_left
float32 angular_speed_right
